src/scripts/run-tests.sh

```diff
@@ -100,7 +94,7 @@ selectByName="{$selectByName}"
     'PDFTestReport','${PARAM_TEST_RESULTS_PDF}',\
     'Strict',${PARAM_STRICT},\
     'UseParallel',${PARAM_USE_PARALLEL},\
-    'SelectByName', $selectByName,\
+    'SelectByName', '${selectByName}',\
```

Comment:
Do we need the intermediate variable any more?
```suggestion
    'SelectByName', '${PARAM_SELECT_BY_NAME}',\
```